### ISSUE TITLE
studio(a11y): colors should have a name description

### ISSUE THREAD

**Issue by donroyco** [deckgo/deckdeckgo#544]
### Affected part of DeckDeckGo
- [x] Studio
- [ ] Remote control
- [ ] Documentation
- [ ] Demo
- [ ] Starter kit
- [ ] Poll
- [ ] Web Components
- [ ] Templates

#### Reproduction

Steps to reproduce:
1. Go to the editor
2. Click on the tab 'Style'
3. Look at the sidebar 
 
#### Expected Behavior

I would expect that the colors in the palette are also written down in order to deal with people with color blindness. According to [National Eye Institute](https://www.nei.nih.gov/learn-about-eye-health/eye-conditions-and-diseases/color-blindness), 1 out of 12 men has color blindness (any form).

#### Actual Behavior

When checking the colors as a non color blind person, everything looks fine.

<img width="445" alt="Colors viewed by non color blind person" src="https://user-images.githubusercontent.com/1763537/70861752-c3ade200-1f32-11ea-9ead-eb651ed0d74d.png">

_Non color blind users will see this._

<img width="414" alt="Colors viewed by color blind person (deuteranopia)" src="https://user-images.githubusercontent.com/1763537/70861779-138ca900-1f33-11ea-90f4-5d370b8332f1.png">

_Color blind person (deuteranopia) will see this._

<img width="422" alt="Colors viewed by color blind person (tritanopia)" src="https://user-images.githubusercontent.com/1763537/70861791-420a8400-1f33-11ea-9a00-f5794f51e169.png">

_Color blind person (tritanopia) will see this._

This little app called [Sim Daltonism](https://apps.apple.com/ca/app/sim-daltonism/id693112260?en) is what I used to simulate the color blindness. As you can see, it's not clear anymore what color every color is.

**Comment by peterpeterparker** [comment]
good point @donroyco 

we are working with rgb color, what do you think, is is better to display a label like "Current color rgba(255, 44, 33, 0.4)" or the hex value?  what would be your suggestion?

thank you in advance for your inputs!

**Comment by donroyco** [comment]
@peterpeterparker For the colors that are already displayed, I think adding the aria value underneath the colors would already help. 

<img width="295" alt="Color descriptions" src="https://user-images.githubusercontent.com/1763537/70865993-397e7180-1f64-11ea-9e46-5da6372a7d07.png">

For the selected color it's hard to define what's best. I don't think a lot of people will know what `rgba(255, 44, 33, 0.4)` means in terms of color, but neither do they in hex value.

**Comment by peterpeterparker** [comment]
thank you for the inputs!

I've to say I'm not sure yet what's the best solution, because users could also select any colors through the "extended" picker and because I also use the component in other apps ;)

I should think about it but of course you are definitely right, the accessibility could/should be improved

**Comment by donroyco** [comment]
Yeah, it's true that people can select any color. Let's see what other people think and what other editors do.

**Comment by peterpeterparker** [comment]
Thx for the inputs @donroyco  

I  implemented it with a new label + hex code. Like "White #ffffff" and if user select a custom color then "Custom #f32f23". I thought that's small win without too much engineering. 

Also as of now, the opacity rate (like 88%) is also displayed.

<img width="1029" alt="Capture d’écran 2020-05-15 à 14 06 59" src="https://user-images.githubusercontent.com/16886711/82049174-31169800-96b6-11ea-9fd7-1d61a046c22d.png">
